api-module/src/main/java/org/devridge/api/security/filter/JwtAuthorizationFilter.java

```diff
@@ -48,8 +48,7 @@ public JwtAuthorizationFilter(AuthenticationManager authenticationManager, Membe
     // 3. DB 에서 리프레시토큰 조회. 리프레시 토큰이 유효하다면 -> 새로운 액세스토큰 발급, 그렇지 않다면 -> 인증된 객체를 저장하지 않고 doFilter
 
     private final List<RequestMatcher> excludedUrlPatterns = Arrays.asList(//이 필터 적용 안할 url 지정
-            new AntPathRequestMatcher("/login/mailConfirm"),
-            new AntPathRequestMatcher("/login/authenticate")
+            new AntPathRequestMatcher("api/email-verifications")
```

Comment:
```suggestion
            new AntPathRequestMatcher("/api/email-verifications")
```
이거 앞에 `/`는 없어도 괜찮은가요?

Reply:
아뇨 있어야 될 거 같아요 ㅎㅎ.. 빼먹은게 많네요